plugins/inputs/ethtool/ethtool.go

```diff
@@ -0,0 +1,61 @@
+package ethtool
+
+import (
+	"sync"
+)
+
+type Command interface {
+	DriverName(intf string) (string, error)
+	InterfaceNames() ([]string, error)
+	Stats(intf string) (map[string]uint64, error)
+}
+
+type Ethtool struct {
+	// This is the list of interface names to include
+	IncludeInterfaces []string `toml:"include_interfaces"`
+
+	// This is the list of interface names to ignore
+	IgnoreInterfaces []string `toml:"ignore_interfaces"`
+
+	// Whether to include the driver name in the tag
+	DriverName bool `toml:"driver_name_tag"`
+
+	// Fields to include - all if empty
+	IncludeFields []string `toml:"include_fields"`
```

Comment:
Remove this, as [`fielddrop`](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#modifiers) can be used to remove unwanted fields.

Reply:
Removed